<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HeartRateScreen</class>
 <widget class="QWidget" name="HeartRateScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget[objectName|=&quot;invisibleButton&quot;]{border: none; }</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>768</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/heart_rate_background.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="constantSelectedLabel">
   <property name="geometry">
    <rect>
     <x>693</x>
     <y>478</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/RadioButton_Selected.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="dynamicSelectedLabel">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>478</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="images.qrc">:/images/images/RadioButton_Selected.png</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="invisibleButton_3">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>440</y>
     <width>131</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="invisibleButton_4">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>440</y>
     <width>121</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="invisibleButton_5">
   <property name="geometry">
    <rect>
     <x>830</x>
     <y>640</y>
     <width>141</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="invisibleButton_6">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>630</y>
     <width>181</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
